code/game/objects/structures/beds_chairs/bed.dm

```diff
@@ -161,7 +161,16 @@
 	anchored = 0
 	buildstacktype = /obj/item/stack/sheet/mineral/wood
 	buildstackamount = 10
+	var/owner = ""
```

Comment:
This isn't actually that bad one dangling reference less for the mob and it doesn't really need anything more.